pulpcore/cli/core/context.py

```diff
@@ -23,6 +23,7 @@ class PulpAccessPolicyContext(PulpEntityContext):
     ID_PREFIX = "access_policies"
 
     def reset(self) -> Any:
+        self.pulp_ctx.needs_plugin(PluginRequirement("core", min="3.17"))
```

Comment:
We are not being consistent in writing the "z" stream parameter when it is zero.

Reply:
I think of fetures to come with a X.Y while bugfixes and workarounds need the specific X.Y.Z but i can take a look and make them all X.Y.Z. WDYT?